Source/Scene/ImageryLayer.js

```diff
@@ -1028,6 +1128,161 @@ define([
         texture : undefined
     };
 
+    var geographicCartographicScratch = new Cartographic();
+    var projectedScratch = new Cartesian3();
+    function reprojectFromArbitrary(command, context, texture, cartographicRectangle, tileRectangleInProjection, projection) {
+        var reproject = {
```

Comment:
What is the point of this object? Is `destroy` supposed to be called in `postExecute`? The framebuffer is automatically destroyed. You can set `persists` to `false` to destroy the shader and vertex array after the command is executed.